src/engine/renderer/Material.cpp

```diff
@@ -1647,11 +1647,12 @@ void MaterialSystem::CullSurfaces() {
 
 		vec4_t frustumPlanes[6];
 		for ( int i = 0; i < 6; i++ ) {
-			VectorCopy( PVSLocked ? lockedFrustum[i].normal : frustum[0][i].normal, frustumPlanes[i] );
-			frustumPlanes[i][3] = PVSLocked ? lockedFrustum[i].dist : frustum[0][i].dist;
+			VectorCopy( PVSLocked ? lockedFrustums[view][i].normal : frustum[0][i].normal, frustumPlanes[i] );
+			frustumPlanes[i][3] = PVSLocked ? lockedFrustums[view][i].dist : frustum[0][i].dist;
 		}
 		matrix_t viewMatrix;
 		if ( PVSLocked ) {
+			VectorCopy( lockedOrigins[view], origin );
```

Comment:
Does locking PVS actually hold the list of views fixed? Otherwise this doesn't make sense.

Reply:
You're right, I added that now.